core/Test/Tasty/Run.hs

```diff
@@ -45,17 +46,17 @@ import Test.Tasty.Runners.Reducers
 data Status
   = NotStarted
     -- ^ test has not started running yet
-  | Executing Progress
```

Comment:
Ah, I forgot there was a Progress inside Status. Was there a reason it couldn't be used?

Reply:
I made an assumption based on the issue referenced in the comments #33 that updating the status TVar too often may cause other threads to wake up and slow things down or misbehave.

I'll give it a try with the status and see if it shakes anything out.

Just as an FYI, I'm using both the core-tests and the test suite for waargonaut to see how things go. Waargonaut has Hedgehog, HUnit, and Golden tests so it felt like a good "stress test"

Reply:
Seems to work just fine! :man_shrugging: I've reverted the addition of a separate `TVar Progress`.